radis/test/lbl/test_factory.py

```diff
@@ -725,6 +725,57 @@ def test_diluents_molefraction(verbose=True, plot=False, *args, **kwargs):
     sf.eq_spectrum(Tgas=300, mole_fraction=0.4)
 
 
+def test_vaex_and_pandas_spectrum():
```

Comment:
add in docstrings what you're testing, also add the link to this PR (e.g. "testing Vaex as implemetned in https://github.com/radis/radis/pull/580"  )

Reply:
add the comments in docstrings under the function, not above. Same for all other test functions